docs/usage/.pages

```diff
@@ -0,0 +1,51 @@
+nav:
+  - Home: 'index.md'
+  - ... | getting-started
+  - Troubleshooting: 'troubleshooting.md'
+  - Configuration:
+      - 'Self-hosted': 'self-hosted-configuration.md'
+      - 'Repository': 'configuration-options.md'
+      - 'Presets': 'config-presets.md'
+  - ... | key-concepts
+  - ... | modules
```

Comment:
On the preview I can't see a ⬇️ to expand the Datasources sidebar.

Reply:
yes, it's a single page needs this 😉 
- #14772